CMakeLists.txt

```diff
@@ -525,12 +525,20 @@ if(QUIC_TLS STREQUAL "openssl")
 endif()
 
 if (QUIC_ENABLE_LOGGING)
-    execute_process(COMMAND clog --installDirectory ${QUIC_BUILD_DIR}/clog)
+    add_subdirectory(submodules/clog)
+
+    if (WIN32)
+        LIST(APPEND CMAKE_PROGRAM_PATH  "${PROJECT_BINARY_DIR}/submodules/clog/artifacts/win-x64")
+        set(CLOG_EXE  "${PROJECT_BINARY_DIR}/submodules/clog/artifacts/win-x64/clog")
+    elseif (UNIX)
+        LIST(APPEND CMAKE_PROGRAM_PATH  "${PROJECT_BINARY_DIR}/submodules/clog/artifacts/linux-x64")
+        set(CLOG_EXE  "${PROJECT_BINARY_DIR}/submodules/clog/artifacts/linux-x64/clog")
+    endif()
```

Comment:
Is it ok to always reference `x64`? I assume this means we become unbuildable on an x86 machine?

Reply:
This also breaks macOS builds. Which is something we do need working.

Reply:
Fixed